### ISSUE TITLE
v0.9: "codeLineKey is duplicated" is occured after same method execution that throws exception to caller

### ISSUE THREAD

**Issue by hiroshitoda** [SahaginOrg/sahagin-java#18]
This issue will affect to some structured test codes.
Very simple sample codes are below:
- SampleTest.java

``` Java
    @Test
    public void test1()
    {
        SamplePage samplePage = new SamplePage(driver);
        assertThat(samplePage.getSampleText(), is("sample1"));
    }
    @Test
    public void test2()
    {
        SamplePage samplePage = new SamplePage(driver);
        assertThat(samplePage.getSampleText(), is("sample2"));
    }
```
- SamplePage.java

``` Java
    public String getSampleText()
    {
        return driver.findElement(By.id("sampleElement")).getText();
    }
```

If `sampleElement` doesn't exist, `test1()` fails with NoSuchElementException(expected,) and `test2()` fails with RuntimeException and "codeLineKey is duplicated"(not expected.)

I'm afraid that `afterCodeLineHook()` is not executed properly in this case.

**Comment by NozomiIto** [comment]
Oh, I missed it.
I've just fixed the problem, and will release the fix soon, thank you!

**Comment by NozomiIto** [comment]
Released [0.9.1](https://github.com/SahaginOrg/sahagin-java/releases/tag/0.9.1) including the fix for this problem.

We don't plan to release version 0.9.1 for Sahagin Jenkins plug-in nor Sahagin-Groovy since this problem does not occur for them and Sahagin-Java 0.9.1 will work with Sahagin Jenkins plugin-in 0.9
